src/components/Item.js

```diff
@@ -0,0 +1,35 @@
+import PropTypes from 'prop-types';
+import { BsArrowLeftCircle } from 'react-icons/bs';
+import { NavLink } from 'react-router-dom';
+
+function Item({
+  name, currency, stockExchange, exchangeShortName, symbol,
+}) {
+  return (
+    <div>
+      <h1>{name}</h1>
+      <p>{currency}</p>
+      <p>{stockExchange}</p>
+      <p>{exchangeShortName}</p>
+      <p>{symbol}</p>
+      <NavLink
+        to="/"
+      >
+        <BsArrowLeftCircle
+          color="white"
+          size={22}
+        />
+      </NavLink>
+    </div>
+  );
```

Comment:
- [ ] Great work on completing your details page! 👏 However, to ensure that your project aligns with the project requirements, there are a few key points to consider:

1. It's crucial to maintain the design layout as specified in the project requirements. When reviewing your details page, it's clear that your design layout doesn't match the provided requirements.
2. Kindly consider moving your `back` button to the top of your page on the navigation as shown on the main design. This will help make it easy for your users to find and also create a good user experience.

    - To bring your project in line with the project requirements, I recommend making the necessary adjustments to ensure your design aligns with the specified layout. This alignment with the provided design is essential to meet the project's objectives. See below for more details:
<table>
<tr>
<th align=“middle”>Behance design</th>
<th align=“middle”>Your design</th>
</tr>
<tr>
<td>
<img width="194" alt="Screenshot 2023-10-21 at 03 54 08" src="https://github.com/alex1779/metrics-webapp/assets/107192076/ebebc517-79a6-4bd6-834f-0eea31013d56">
</td>
<td>
<img width="337" alt="Screenshot 2023-10-21 at 03 35 22" src="https://github.com/alex1779/metrics-webapp/assets/107192076/d43ef413-3d21-4a1e-ab8a-7068804c4246">
</td>
</tr>
</table>

Happy coding!😊